tests/mock/select-certificate-script-EST-IDEMIA.hpp

```diff
@@ -160,7 +160,10 @@ const PcscMock::ApduScript ESTEID_IDEMIA_V1_SELECT_AUTH_CERTIFICATE_AND_AUTHENTI
       0xff, 0xff},
      {0x90, 0x00}},
 
-    // Internal auth
+// Internal authenticate.
+#ifdef NO_SIGNATURE
```

Comment:
Did you use this `#ifdef` during testing? Do we need to keep it?

Reply:
Yes, its there for same reason as gemalto use case